netci.groovy

```diff
@@ -28,30 +25,35 @@ platformList.each { platform ->
     def buildCommand = '';
     def osForGHTrigger = os
     def version = "latest-or-auto"
+    def dockerRepository = "microsoft/dotnet-buildtools-prereqs"
+    def dockerContainer = ''
+    def dockerWorkingDirectory = "/src/core-setup"
+    def dockerCommand = ''
+    def portableArgs = ''
 
     // Calculate build command
     if (os == 'Windows_NT') {
         buildCommand = ".\\build.cmd -ConfigurationGroup=${configuration} -TargetArchitecture=${architecture}"
+        if ((architecture == 'arm' || architecture == 'arm64')) {
```

Comment:
I think we should simply switch all our CI builds to portable, independent of architecture.

CC @eerhardt.